### PR TITLE
#16

### PR DESCRIPTION
**Pull request by kuntzer**
See [Issue 16](https://github.com/mtewes/tenbilac/issues/16)

### CODE REVIEW

**Comment by mtewes** [comment]
@kuntzer, did you see these comments ?

**Comment by kuntzer** [comment]
What comments?

**Comment by mtewes** [comment]
Those just above, on this page
https://github.com/mtewes/tenbilac/pull/17#issuecomment-264120536

File: tenbilac/train.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class Training:
 	"""
 
 	
-	def __init__(self, net, dat, errfctname="msrb", itersavepath=None, autoplotdirpath=".", autoplot=False, verbose=False, name=None):
+	def __init__(self, net, dat, errfctname="msrb", itersavepath=None, autoplotdirpath=".", autoplot=False, verbose=False, name=None, inames=None, onames=None):
```
Comment (kuntzer) [inline tenbilac/train.py]: Good point. I changed that.

**Pull request merged**